v1 Update Content to Docs

joshbermanssw opened the pull request:
v1 of content

isaaclombardssw reviewed (approved):
Approving and leaving fixes up to you 👍

Nice work

joshbermanssw commented:
> This is really solid...
> 
> Only a couple things:
> 
> 1. probs not relevant to this PR, but this the contrast makes things a little hard to see

  Fixed 

> 2. Links go to google

   fixed

> 3. Is this TODO still TBC?

isaaclombardssw reviewed (dismissed):
✅
